packages/addons/addon-depends/pcsc-lite/package.mk

```diff
@@ -17,21 +17,19 @@
 ################################################################################
 
 PKG_NAME="pcsc-lite"
-PKG_VERSION="1.8.16"
+PKG_VERSION="1.8.22"
 PKG_ARCH="any"
 PKG_LICENSE="GPL"
 PKG_SITE="http://pcsclite.alioth.debian.org/pcsclite.html"
-PKG_URL="https://alioth.debian.org/frs/download.php/file/4164/${PKG_NAME}-${PKG_VERSION}.tar.bz2"
+PKG_URL="https://alioth.debian.org/frs/download.php/latestfile/39/${PKG_NAME}-${PKG_VERSION}.tar.bz2"
```

Comment:
This should be changed to

PKG_URL="https://alioth.debian.org/frs/download.php/file/${PKG_VER}/${PKG_NAME}-${PKG_VERSION}.tar.bz2"